src/components/PrefectureTable/PrefectureTable.js

```diff
@@ -131,12 +131,14 @@ const drawPrefectureTable = (prefectures, totals, prefectureTrendCharts) => {
         <td class="count">${pref.deceased ? pref.deceased : ""}</td>
         <td class="count">${pref.active || ""}</td>
         </tr>`;
-      prefectureTrendCharts = drawPrefectureTrend(
-        `#Unspecified-trend`,
-        pref.dailyConfirmedCount,
-        maxConfirmedIncrease,
-        prefectureTrendCharts
-      );
+      setTimeout(() => {
```

Comment:
FYI: Using `setTimeout` like this is basically a hack to create a new macrotask into the queue.

Reply:
Optional suggestion: This can be a little more readable if you wrap the setTimeout() call in another function called something like enqueueMacroTask(). That would mean the call would look like :

```
const enqueueMacroTask = (fn) => {
  setTimeout(fn, 0);
}

... 
enqueueMacroTask(() => { 
   some work here
})
```

That way when you read the code you don't have to guess why there's a setTimeout here.